README.md

```diff
@@ -94,6 +94,59 @@ Signed-Releases: Fail 10
 Signed-Tags: Fail 10
 ```
 
+### npm
```

Comment:
Maybe move to sub-heading of "Package manager support"

Reply:
👍